crates/sui-source-validation/src/tests.rs

```diff
@@ -21,29 +21,29 @@ use sui_types::{
 };
 use test_cluster::TestClusterBuilder;
 
-use crate::{BytecodeSourceVerifier, SourceMode};
+use crate::{BytecodeSourceVerifier, SourceMode, CURRENT_COMPILER_VERSION};
 
 #[tokio::test]
 async fn successful_verification() -> anyhow::Result<()> {
     let mut cluster = TestClusterBuilder::new().build().await;
     let context = &mut cluster.wallet;
 
+    let b_ref_fixtures = tempfile::tempdir()?;
     let b_ref = {
-        let fixtures = tempfile::tempdir()?;
-        let b_src = copy_published_package(&fixtures, "b", SuiAddress::ZERO).await?;
+        let b_src = copy_published_package(&b_ref_fixtures, "b", SuiAddress::ZERO).await?;
```

Comment:
This pattern refactors tests to preserve package temp dirs so that we can find and resolve package roots of dependencies from compiled units, e.g., above `SourcePackageLayout::try_find_root(&local_unit.source_path)?;`